worlds/stardew_valley/data/recipe_source.py

```diff
@@ -94,6 +94,14 @@ def __repr__(self):
         return f"SkillSource at level {self.level} {self.skill}"
 
 
+class SkillCraftsanitySource(SkillSource):
+    def __init__(self, skill: str, level: int):
+        super().__init__(skill, level)
+

```

Comment:
`__init__()` does not need to be overridden if it is not changing anything compared to the superclass's `__init__()`.

```suggestion
```